Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingImageBufferBackend.mm

```diff
@@ -91,6 +91,8 @@ static CFDictionaryRef makeContextOptions(const DynamicContentScalingImageBuffer
 {
 }
 
+DynamicContentScalingImageBufferBackend::~DynamicContentScalingImageBufferBackend() = default;
```

Comment:
Why is this needed?

Reply:
Will check if it still is; this was to avoid adding a header to the .h

Reply:
it is not